src/device/usbd.c

```diff
@@ -543,9 +548,18 @@ static bool process_control_request(uint8_t rhport, tusb_control_request_t const
 
           dcd_set_config(rhport, cfg_num);
```

Comment:
for this to work: `dcd_set_config(rhport, 0)` must be implemented by all DCD to reset all the opened endpoints when confignum is 0